.circleci/config.yml

```diff
@@ -60,12 +60,19 @@ commands:
           rust-version: <<parameters.rust-version>>
       - run:
           name: Test
-          command: cargo test --all --verbose -- --nocapture
+          command: GLEAN_TEST_COVERAGE=$(realpath glean_coverage.txt) cargo test --all --verbose -- --nocapture
```

Comment:
Have to use `realpath` here since `cargo test` changes the working directory.  This makes sure there is a single file (even though there are multiple crates tested) and it ends up where we expect it to.